src/engine/controls/cuecontrol.cpp

```diff
@@ -2218,16 +2122,15 @@ void CueControl::slotLoopUpdated(double startPosition, double endPosition) {
         return;
     }
 
-    if (!m_pLoadedTrack) {
-        return;
-    }
-
     if (m_pCurrentSavedLoopControl->getStatus() != HotcueControl::Status::Active) {
         slotLoopReset();
         return;
     }
 
-    CuePointer pCue(m_pCurrentSavedLoopControl->getCue());
+    CuePointer pCue = m_pCurrentSavedLoopControl->getCue();
+    if (!pCue) {
```

Comment:
This was never checked before. Could this ever become a nullptr? If not then use VERIFY_OR_DEBUG_ASSERT.